src/modal/index.unit.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import Modal from 'modal'
+import Button from 'button'
+
+describe('<Modal>', () => {
+  it('Should have default props with appropriate values and content', () => {
+    const wrapper = mount(<Modal isOpen={true}><p>My modal</p></Modal>)
+    expect(wrapper.prop('fullscreen')).toEqual(false)
```

Comment:
if this is a plain value, use `toBe`. Here you can use an object to test all of the props, and use `toEqual`

Reply:
Done